### PR TITLE
Added some minor perf improvements, a11y and best practices

### PR DESCRIPTION
**Pull request by bovas85**
Please have a look mate :)

### CODE REVIEW

File: src/components/FormButton.vue [hunk 1]
```diff
@@ -35,20 +35,20 @@ export default {
   padding: var(--grid-two) var(--grid-four);
   border: 0;
   margin: var(--grid-three) 0;
-  background-color: var(--color-green);
+  background-color: var(--color-green--dark);
```
Comment (blake-newman) [inline src/components/FormButton.vue]: What is the reasons for changing this?

File: src/pages/privacy-policy.vue [hunk 2]
```diff
@@ -103,51 +103,38 @@
...
-        <list-item>
-          To gather analysis or valuable information so that we can improve our Service
-        </list-item>
-        <list-item>
-          To monitor the usage of our Service
-        </list-item>
-        <list-item>
-          To detect, prevent and address technical issues
-        </list-item>
+        <list-item>To provide customer support</list-item>
```
Comment (blake-newman) [inline src/pages/privacy-policy.vue]: When in-lining prettier does some wacky stuff... It's pretty annoying. Could we revert these changes as it breaks the IDE syntax highlighting

**Comment by blake-newman** [comment]
Thanks for the awesome contribution, adds everything i should have done myself. 

Created a PR into your PR, so that you can see the changes i am requesting without making you do the work. Let me know what you think: https://github.com/bovas85/website/pull/1

File: src/components/FormButton.vue (hunk 1 above)
Comment (bovas85) [reply src/components/FormButton.vue]: the contrast was not AA accessible :)

File: src/pages/privacy-policy.vue (hunk 2 above)
Comment (bovas85) [reply src/pages/privacy-policy.vue]: it does indeed :D will revert

**Comment by bovas85** [comment]
Added your PR into this, thanks!

File: src/pages/privacy-policy.vue (hunk 2 above)
Comment (bovas85) [reply src/pages/privacy-policy.vue]: reverted

File: src/components/FormButton.vue (hunk 1 above)
Comment (blake-newman) [reply src/components/FormButton.vue]: Thought that was the case :)
